pkg/apis/options/alpha_options.go

```diff
@@ -12,7 +12,7 @@ type AlphaOptions struct {
 	// Upstreams is used to configure upstream servers.
 	// Once a user is authenticated, requests to the server will be proxied to
 	// these upstream servers based on the path mappings defined in this list.
-	Upstreams Upstreams `json:"upstreams,omitempty"`
+	Upstreams UpstreamConfig `json:"upstreams,omitempty"`
```

Comment:
I think the naming would be good to keep consistent. My last comment was primarily a UX based thing, so I'm thinking that in the yaml it would be good to be clear
```
usptreamConfig:
  proxyRawPath: true
  upstreams:
  - type: ...
```


Does that make sense? 
```suggestion
	UpstreamConfig UpstreamConfig `json:"upstreamConfig,omitempty"`
```